include/nbl/video/IPhysicalDevice.h

```diff
@@ -358,6 +358,7 @@ class NBL_API2 IPhysicalDevice : public core::Interface, public core::Unmovable
 
             SFormatImageUsage(core::bitflag<asset::IImage::E_USAGE_FLAGS> usages)
                 : isInitialized(1), 
```

Comment:
`isInitialized` is still a redundant member

Reply:
yea you put a TODO to remove. it's stil there. I'll probably get to it